app/models/fe/state_chooser.rb

```diff
@@ -3,8 +3,8 @@
 # - drop down of states
 module Fe
   class StateChooser < Question
-    def choices(country = 'US')
-      @states = Carmen.states(country)
+    def choices(locale = nil, country = 'US')
```

Comment:
Just added that.  We never call the state chooser for another language, so far that's been left to the apps.  We do it in Canada so I'll have to update it.  I came across this problem originally because I had switched over all choices(...) calls to choices(session[:locale], ...) in FE without realizing this method didn't change.  We could just keep it not translated.. or let the view translate it..